src/auro-badge.js

```diff
@@ -24,15 +25,17 @@ import styleCss from "./style-css.js";
 
 // build the component class
 class AuroBadge extends LitElement {
-  // constructor() {
-  //   super();
-  // }
+  constructor() {
+    super();
+    this.dom = new DOMParser().parseFromString(closeIcon.svg, 'text/html');
```

Comment:
Would like to replace with auro-icon, but it seems like there's a bug with transpiling

Reply:
If there is really only a single use case for an icon or two, I'd prefer that we just make it a straight dependency. That way there are no loose ends and surprises down the road.